tt_metal/jit_build/build.hpp

```diff
@@ -96,7 +95,6 @@ class JitBuildState {
     void compile_one(const string& log_file, const string& out_path, const JitBuildSettings *settings, const string& src, const string &obj) const;
     void link(const string& log_file, const string& out_path) const;
     void elf_to_hex8(const string& log_file, const string& out_path) const;
-    void hex8_to_hex32(const string& log_file, const string& out_path) const;
```

Comment:
why is the new one no longer a member function?

Reply:
reverted